src/Command/Revert.php

```diff
@@ -28,9 +27,9 @@ class Revert extends AbstractCommand
     const NAME = 'revert';
 
     /**
-     * List of quality patches to revert.
+     * List of patches to revert.
      */
-    const ARG_QUALITY_PATCHES = 'quality-patches';
+    const ARG_LIST_OF_PATCHES = 'list_of_patches';
```

Comment:
Cloud Tools use hyphens instead of underscores

Reply:
Will be fixed, thanks

Reply:
@shiftedreality fixed, please check